project/Assets/Player/PlayerMovement.cs

```diff
@@ -26,7 +26,7 @@ void Start()
 
     void Update()
     {
-        if (Input.GetKeyDown(KeyCode.UpArrow) && bird.IsAwake())
```

Comment:
why did you removed this? Does this have influence on the menu logic?

Reply:
i was just reverting another change i made there in a prior commit that shouldn't have been made either as it also had no influence on the issue i was working on at the time
the commit in question: https://github.com/marcos-exaud/flappy-bird-2/commit/a86f8c41c5c0096437c84903d3f024ced165ae70#diff-544686eea27d56849e6da6e8a7d7151c4853e64154ab908ca6a3df989a8c3dc5